### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: tokenizer/test4.test [hunk 1]
```diff
@@ -438,8 +438,8 @@
     { "code": "invalid-character-sequence-after-doctype-name", "line": 1, "col": 16 }
 ]},
 
-{"description":"Doctype sistem",
-"input":"<!DOCTYPE html SISTEM \"AbC\">",
```
Comment (Ms2ger) [inline tokenizer/test4.test:442]: Do we want to change the actual test here? It's probably not a bad idea to keep testing typos.

Comment (gsnedders) [reply tokenizer/test4.test:442]: And it looks like it's specifically trying to test this behaviour with this typo?

Comment (luzpaz) [reply tokenizer/test4.test:442]: i was wondering that. I can revert these if so.

Comment (annevk) [reply tokenizer/test4.test:442]: Please revert changes to test inputs.

**Comment by luzpaz** [comment]
I can squash changes or leave them for review convenience ?

**Review by annevk — APPROVED** [review]

**Comment by annevk** [comment]
I'll squash when landing, thanks for following up!

**Pull request merged**